src/MutatorOrchestrator.ts

```diff
@@ -28,24 +29,22 @@ export default class Mutator {
    * @returns {Mutant[]} The generated Mutants.
    */
   mutate(sourceFiles: string[]) {
```

Comment:
we should maybe rename this to `generateMutants`. I think that name more accurately describes what the method is doing.